Bookish/Views/Book/Catalogue.cshtml

```diff
@@ -19,31 +27,56 @@
             <th>Number of available copies</th>
             <th>Number of searches</th>
         </tr>
-        
+        </thead>
+
+        <tbody>
         @{
+            int i = -1;
+            Book btnBook = new Book();
```

Comment:
I don't think you should need to create any new `Book` models in the frontend like this